.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: go
+
+go:
+  - "1.9.7"
+  - "1.10.x"
+  - "1.11.x"
+  - "1.12.x"
+  - "1.x"
+  - master
+  
+os:
+  - linux
+  - osx
+
+cache:
+  directories:
+    - $HOME/.cache/go-build
+    - $HOME/gopath/pkg/mod
+    
+# Skip the installation step. Don't "go get" dependencies.
+install: true
+
+env:
+  global:
+    - GO111MODULE=on
+
+# Only clone the most recent commit.
+git:
+  depth: 1
+
+before_script:
+  # needed until we stop supporting 1.10.x
+  GO111MODULE=off;
+  go get -t -d ./...;   
+
+script:
+  - ./.travis.gofmt.sh
```

Comment:
Wondering if we need this to be a separate script? Looks like it's common practice to have it separate like this so probably fine.

Reply:
I agree it would be nice to have one less file, but seems to be common practice in the google repositories